tests/testthat/test-5makePhyloseqFromTreeSummarizedExperiment.R

```diff
@@ -0,0 +1,47 @@
+context("makePhyloseqFromTreeSummarizedExperiment")
+
+test_that("makePhyloseqFromTreeSummarizedExperiment", {
+
+    skip_if_not_installed("phyloseq")
+
+    # TSE object
+    data(GlobalPatterns)
+    tse <- GlobalPatterns
+
+    phy <- makePhyloseqFromTreeSummarizedExperiment(GlobalPatterns)
+
+    # Test that assay is in otu_table
+    testthat::expect_equal(as.data.frame(phyloseq::otu_table(phy)@.Data), as.data.frame(assays(tse)$counts))
+
+    # Test that rowData is in tax_table
+    testthat::expect_equal(as.data.frame(phyloseq::tax_table(phy)@.Data), as.data.frame(rowData(tse)))
+
+    # Test that colData is in sample_table
+    testthat::expect_equal(phyloseq::sample_data(phy),
+                           phyloseq::sample_data(data.frame(colData(tse))))
+
+    # Test that rowTree is in phy_tree
+    testthat::expect_equal(phyloseq::phy_tree(phy), rowTree(tse))
+
+    # Test that referenceSeq is in refseq. There is no referenceseq in TSE, so
+    # referenceseq slot is not made in the phyloseq object
+    #testthat::expect_equal(phyloseq::refseq(phy), referenceSeq(tse))
```

Comment:
The referenceSeq slot is present, but the value `NULL`

Reply:
Now the function has new structure.

So, if tse's referenceSeq has value NULL, phyloseq's refseq should be also NULL? 

Now the function does not add NULLs, because e.g. 'phyloseq' package's 'GlobalPatterns' data does not have NULL in refseq even though the corresponding tse have NULL.

Reply:
The comment was a comment on the comment 😄 

`TSE` has a `referenceSeq` slot, which by default is `NULL`. So, yes, if `is.null(referenceSeq(tse))`, the `phyloseq` object also shouldn't have sequence data. How `phyloseq` internally stores the sequence, if have never bother to look up.

Reply:
Oh, okay, that explains...